pysass.cpp

```diff
@@ -414,13 +555,14 @@ PySass_compile_string(PyObject *self, PyObject *args) {
     Sass_Output_Style output_style;
     int source_comments, error_status, precision, indented;
     PyObject *custom_functions;
+    PyObject *custom_importers;
     PyObject *result;
-
+    
```

Comment:
Heh, we diverge on style guides in a few interesting ways, and yeah, "redspace" is one of them.  I will, of course, ensure my changes conform to the local package style in the end.